include/solid_mechanics/mf_nh_operator.h

```diff
@@ -871,10 +871,18 @@ NeoHookOperator<dim, Number>::do_operation_on_cell(FECellIntegrator &phi) const
                       {
                         VectorizedArrayType sum =
                           inv_jac[e][0] *
+#if DEAL_II_VERSION_GTE(9, 6, 0)
+                          ref_grads[(d * n_q_points + q) * dim + 0];
+#else
```

Comment:
@davidscn  Is the original code also affected? If yes could you create a PR there as well?

Reply:
I was not sure whether this code is currently maintained..